<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDirTrabajos</class>
 <widget class="QDialog" name="frmDirTrabajos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seleccione un directorio para almacenar los trabajos</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione un directorio para almacenar datos de los trabajos.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Debe ser un directorio donde tenga permiso de escritura.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>La ruta al directorio se almacenará en el archivo dirTrabajos.txt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Se intentará guardar este archivo en el directorio base donde se guarda este programa,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>si no se consigue, se intentará guardar en c:/, y si esto también falla, si no desea</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>que aparezca este cuadro siempre que ejecute el programa,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>, tendrá que crear el archivo c:/dirTrabajos.txt e introducir la ruta usted mismo.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>La ruta a los trabajos no debe contener acentos, ni eñes, ni espacios, y debe tener</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>531</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>la barra de dividir para separar las carpetas. Por ejemplo c:/Trabajos/delProp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Directorio para los trabajos:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbDirTrabajos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bttSeleccionar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttAceptar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
